idea-plugin/build.gradle

```diff
@@ -22,9 +22,10 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$KOTLIN_VERSION"
   implementation "org.assertj:assertj-core:$ASSERTJ_VERSION"
   implementation "org.celtric.kotlin:kotlin-html:$KOTLIN_HTML_VERSION"
-  api project(path: ':compiler-plugin', configuration: 'shadow')
+  compileOnly project(path: ':compiler-plugin', configuration: 'shadow')
```

Comment:
`shadow` plugin doesn't add `classifier` to the dependency in the POM file with `maven-publish` plugin so that's the reason why the dependency is added as `compileOnly` in order not to appear in the wrong way